core/primitives/src/shard_layout.rs

```diff
@@ -128,6 +136,13 @@ pub fn account_id_to_shard_id(account_id: &AccountId, shard_layout: &ShardLayout
     }
 }
 
+pub fn account_id_to_shard_uid(account_id: &AccountId, shard_layout: &ShardLayout) -> ShardUId {
```

Comment:
I wonder why this is not a method on ShardLayout, seems natural

Reply:
That's a very good point. I think I just extended from the original `account_id_to_shard_id` function and didn't think too much.